website/src/pages/roadmap.md

```diff
@@ -61,8 +64,10 @@ Next major release : [0.13.0](https://issues.apache.org/jira/projects/HUDI/versi
 |-----------------------------------------------------------------------------------------------------|--------------|-----------------------------------------------------------------------------------------------------------------------------------------|
-| Protobuf source for DeltaStreamer                                                                   | 0.13.0       | [RFC-57](https://github.com/apache/hudi/blob/master/rfc/rfc-57/rfc-57.md), [HUDI-4399](https://issues.apache.org/jira/browse/HUDI-4399) |
-| Improving Hudi CLI features and usability                                                           | 0.13.0       | [HUDI-1388](https://issues.apache.org/jira/browse/HUDI-1388)                                                                            |
+| Improving Hudi CLI features and usability                                                           | 0.14.0       | [HUDI-1388](https://issues.apache.org/jira/browse/HUDI-1388)                                                                            |
+| Partition TTL management                                                                            | 1.0.0        | [RFC-65](https://github.com/apache/hudi/pull/8062)                                                                         |
+| Diagnostic Reporter                                                                                 | 1.0.0        | [RFC-62](https://github.com/apache/hudi/pull/6600)                                                                         |
 | Hudi integration with Snowflake                                                                     | 1.0.0 onward | [RFC-41](https://github.com/apache/hudi/pull/4074), [HUDI-2832](https://issues.apache.org/jira/browse/HUDI-2832)                        |
 | Support for reliable, event based ingestion from cloud stores - GCS, Azure and the others           | 1.0.0 onward | [HUDI-1896](https://issues.apache.org/jira/browse/HUDI-1896)                                                                            |
-| Mutable, Transactional caching for Hudi Tables   (could be accelerated based on community feedback) | 1.0.0 onward | [Strawman design](https://docs.google.com/presentation/d/1QBgLw11TM2Qf1KUESofGrQDb63EuggNCpPaxc82Kldo/edit#slide=id.gf7e0551254_0_5)    |
\ No newline at end of file
+| Mutable, Transactional caching for Hudi Tables   (could be accelerated based on community feedback) | 1.0.0 onward | [Strawman design](https://docs.google.com/presentation/d/1QBgLw11TM2Qf1KUESofGrQDb63EuggNCpPaxc82Kldo/edit#slide=id.gf7e0551254_0_5)    |
+| Hudi Reverse streamer                                                                               | 1.0.0 onward | [RFC-70](https://github.com/apache/hudi/pull/9040)                                                                                       | 
```

Comment:
We also discussed to include perf improvement for spark-sql writes as a line item. can we add that in

Reply:
it's under execution engine- should i move it elsewhere?